app/controllers/api/service_dialogs_controller.rb

```diff
@@ -3,6 +3,7 @@ class ServiceDialogsController < BaseController
     before_action :set_additional_attributes, :only => [:index, :show]
 
     CONTENT_PARAMS = %w[target_type target_id resource_action_id].freeze
+    ORCHESTRATION_TEMPLATE_DIALOG_REQUIRED_ATTRS = %w[ot_id label].freeze
```

Comment:
probably enough is `ORCHESTRATION_TEMPLATE_DIALOG_ATTRIBUTES` (without REQUIRED)